<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileView</class>
 <widget class="QWidget" name="ProfileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <widget class="QWidget" name="subscriptionHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:#FACA7F;
border-radius: 10px;
}</string>
   </property>
   <widget class="QLabel" name="lblSubsLogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_subscription_black.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblSubscription">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>14</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#000000;
}</string>
    </property>
    <property name="text">
     <string>Subscription</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblExpiry">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#000000;
}</string>
    </property>
    <property name="text">
     <string>101 days until expire</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubscription">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;
border-color:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="topHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>180</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}</string>
   </property>
   <widget class="QWidget" name="settinhgsHolder" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-color:transparent;
}</string>
    </property>
    <widget class="QLabel" name="lblSettingsLogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../assets/assets.qrc">:/img/images/icon_settings.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblSettings">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>2</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#FFFFFF;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:transparent;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="deviceHolder" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-color:transparent;
}</string>
    </property>
    <widget class="QLabel" name="lblDeviceLogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../assets/assets.qrc">:/img/images/icon_devices.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblDevice">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>2</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#FFFFFF;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Devices</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDevice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:transparent;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="referHolder" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-color:transparent;
}</string>
    </property>
    <widget class="QLabel" name="lblReferLogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../assets/assets.qrc">:/img/images/icon_referrals.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblRefer">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>2</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#FFFFFF;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Referrals</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:transparent;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnLogout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#FACA7F;
color: #000000;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QWidget" name="bottomHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}</string>
   </property>
   <widget class="QWidget" name="supportHolder" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-color:transparent;
}</string>
    </property>
    <widget class="QLabel" name="lblSupportLogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../assets/assets.qrc">:/img/images/icon_support.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblSupport">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>2</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#FFFFFF;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Support</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSupport">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:transparent;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="aboutHolder" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-color:transparent;
}</string>
    </property>
    <widget class="QLabel" name="lblAboutLogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../assets/assets.qrc">:/img/images/icon_about.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblAbout">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>2</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#FFFFFF;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>About us</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAbout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:transparent;
border-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
